zipkin-storage/elasticsearch-http/src/test/java/zipkin/storage/elasticsearch/ElasticsearchV5HttpTest.java

```diff
@@ -13,19 +13,18 @@
  */
 package zipkin.storage.elasticsearch;
 
+import java.io.IOException;
 import org.junit.ClassRule;
 import org.junit.experimental.runners.Enclosed;
 import org.junit.runner.RunWith;
 import zipkin.storage.SpanStoreTest;
 
-import java.io.IOException;
-
 @RunWith(Enclosed.class)
 public class ElasticsearchV5HttpTest {
 
   @ClassRule
   public static LazyElasticsearchHttpStorage storage =
-      new LazyElasticsearchHttpStorage("elasticsearch:5.0.1");
+      new LazyElasticsearchHttpStorage("openzipkin/zipkin-elasticsearch5:1.16.1");
```

Comment:
If the version is consistent across all images under openzipkin then we can extract some base class for all TestContainers-based storages and just pass an image name to it. Base class will append version stored as a private constant inside it.

WDYT?